yaml.go

```diff
@@ -307,24 +308,75 @@ type yamlError struct {
 	err error
 }
 
+func (e *yamlError) Unwrap() error {
+	return e.err
+}
```

Comment:
Now that @ccoVeille mentioned this, this method appears to be unused.

Reply:
But `yamlError` is handled by `recover()` in `handleErr` (which accesses the field directly), and then thrown away.